IdentityService/appsettings.json

```diff
@@ -6,7 +6,13 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=ENDAUTODEM4RTK0;Database=TestDbContext;Trusted_Connection=True;TrustServerCertificate=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=ENDAUTOLWBINJTN;Database=TestDbContext;Trusted_Connection=True;TrustServerCertificate=True;MultipleActiveResultSets=true"
+  },
+  "RouteSettings": {
```

Comment:
Please apply the camelCase naming convention for the endpoints discussed earlier so that we can adjust is shortly on API side